### ISSUE TITLE
Add ExcelRtdServer batch update

### ISSUE THREAD

**Issue by govert** [Excel-DNA/ExcelDna#104]
We should add a method to the RTD server implementation that allows a batch update, so that multiple topics can be updated atomically:

public void UpdateValues(Topic[] topics, object] values) {...}

**Comment by govert** [comment]
This is implemented for version 0.34

**Comment by cubitus81** [comment]
Hi Govert,

I have a question regarding this feature. My add-in can have 300 updates in a second. Right now, I called ExcelRtdServer.Topic.UpdateValue for each of the update. I noticed that Excel took approximately 2 seconds (after the last ExcelRtdServer.Topic.UpdateValue is called) to display the latest values  . 

I also tried using ExcelRtdServer.UpdateValues to update 300 values at the same time. Excel still takes 2 seconds (after ExcelRtdServer.UpdateValues is called)  to display all updates. 

My question is, would it still be better to use ExcelRtdServer.UpdateValues (maybe Excel will be less stressed when we send in batches of updates)? 

Thank you.
